plugins/catalog-backend/src/tests/performance/stitchingPerformance.test.ts

```diff
@@ -143,7 +143,7 @@ class Tracker {
 
 describePerformanceTest('stitchingPerformance', () => {
   const databases = TestDatabases.create({
-    ids: [/* 'MYSQL_8', */ 'POSTGRES_16', 'POSTGRES_12', 'SQLITE_3'],
+    ids: [/* 'MYSQL_8', */ 'POSTGRES_17', 'POSTGRES_13', 'SQLITE_3'],
```

Comment:
If there's no MySQL support should we just remove this?

Reply:
@Rugvip wdyt?